### DISCUSSION TITLE
服务器状态查看插件 支持 QQ适配器吗？

### DISCUSSION THREAD

**Discussion by YangRucheng** [nonebot/discussions#545] [问答] [answered]
`nonebot.adapters.qq.exception.ActionFailed: <ActionFailed: 501, >`

应该是支持的 `supported_adapters=None,`

![image](https://github.com/user-attachments/assets/aef13eee-9e07-4468-923d-f05b27ec1dcf)

**Comment by yanyongyu** [comment]
这个报错看上去和nb无关，是QQ接口返回了服务错误，你再试试呢？

**Reply by YangRucheng** [reply]
> 这个报错看上去和nb无关，是QQ接口返回了服务错误，你再试试呢？

我试过了。刚才换了个机器人ID、服务器IP 又试了，还是相同的错误，很奇怪。

![image](https://github.com/user-attachments/assets/8917bdb5-e1f7-41b3-aa90-e4cedc090ea5)

**Reply by yanyongyu** [reply, marked as answer]
可能是你的status文字里面含有什么特殊字符导致接口失败了，你试试改status的模板呢？或者打印一下actionfailed的body

**Reply by YangRucheng** [reply]
应该是 Disk 里出现了特殊字符，我在容器里运行时会出现。

感谢您的帮助 ๑ŐεŐ๑
